src/main/java/Duke.java

```diff
@@ -1,10 +1,161 @@
+
+    }
+
+
+    public static void chat() throws IOException {
+        System.out.println("____________________________________________________________");
+        String str = readCommand();
+        System.out.println("____________________________________________________________");
+
+        if (str.equals("bye")) {
+            bye();
+        } else if (str.equals("add task")) {
+            System.out.println("____________________________________________________________");
+            System.out.println("Please add no more than 100 tasks");
+            System.out.println("____________________________________________________________");
+            addTaskScreen();
+        } else if (str.equals("list")) {
+            listTask();
+            chat();
+        }else {
+            System.out.println(str);
+            System.out.println("____________________________________________________________");
+            chat();
+        }
+    }
+
+
+    public static void addTaskScreen() throws IOException {
+        System.out.println("____________________________________________________________");
+        System.out.println("Please add tasks");
+        System.out.println("____________________________________________________________");
+        String str = readCommand();
+        if (str.equals("bye")) {
+            bye();
+        } else if (str.contains("done")) {
+            System.out.println("Mark task done");
+            markDone(str);
+            addTaskScreen();
+        } else if (str.equals("list")) {
+            listTask();
+            addTaskScreen();
+        } else if ((str.contains("todo"))) {
+            int dividerPosition = str.indexOf(" ");
+            String task = str.substring(dividerPosition + 1);
+            ToDos newTask = new ToDos(task);
+            addTask(newTask);
+            addTaskScreen();
+        } else if ((str.contains("deadline"))) {
+            int dividerPosition = str.indexOf(" ");
+            int dividerPositionSlash = str.indexOf("/");
+            String task = str.substring(dividerPosition + 1 , dividerPositionSlash);
+            String by = str.substring(dividerPositionSlash + 4);
+            Deadlines newTask = new Deadlines(task,by);
+            addTask(newTask);
+            addTaskScreen();
+        } else if ((str.contains("event"))) {
+            int dividerPosition = str.indexOf(" ");
+            int dividerPositionSlash = str.indexOf("/");
+            String task = str.substring(dividerPosition + 1 , dividerPositionSlash);
+            String calender  = str.substring(dividerPositionSlash+4);
+            int dateAndTime = calender.indexOf(" ");
+            String date  = calender.substring(0, dateAndTime);
+            String time  = calender.substring(dateAndTime+1);
+            Events newTask = new Events(task,date,time);
+            addTask(newTask);
+            addTaskScreen();
+        }
```

Comment:
Perhaps, you can have a separate method(s) to manage the extraction of the task other details within the string?